### PR TITLE
SYL-4167: UI cleanup

### PR DESCRIPTION
**Pull request by bartek-sek**
credit_memo
credit_memos
refunds

### CODE REVIEW

File: config/app/twig_hooks/admin/credit_memos/show.yaml
Comment (GSadee) [inline config/app/twig_hooks/admin/credit_memos/show.yaml:1]: To be consistent with for example Sylius/Sylius, the directory should be named `credit_memo`, not `credit_memos`

File: config/app/twig_hooks/admin/credit_memos/show.yaml [hunk 1]
```diff
@@ -0,0 +1,252 @@
+sylius_twig_hooks:
+    hooks:
+        'sylius_refund.admin.credit_memos.show':
```
Comment (GSadee) [inline config/app/twig_hooks/admin/credit_memos/show.yaml]:
```suggestion
        'sylius_refund.admin.credit_memo.show':
```
and the same in other hooks

File: config/app/twig_hooks/admin/credit_memos/show.yaml [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+                priority: 100
+            actions:
+                template: '@SyliusRefundPlugin/admin/credit_memos/show/content/header/title_block/actions.html.twig'
+                priority: 0
+
+        'sylius_refund.admin.credit_memos.show.content.header.title_block.actions':
+            download:
+                template: '@SyliusRefundPlugin/admin/credit_memos/show/content/header/title_block/actions/download.html.twig'
+                enabled: '%sylius_refund.pdf_generator.enabled%'
+                priority: 100
```
Comment (GSadee) [inline config/app/twig_hooks/admin/credit_memos/show.yaml]:
```suggestion
                priority: 0
```
🤔

File: templates/admin/order/refund/content/sections/form/items/table/body/products/refund_value.html.twig [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{% set order = hookable_metadata.context.order %}
+{% set unit = hookable_metadata.context.unit %}
+
+<td class="partial-refund">
+    {% set inputName = "sylius_refund_units["~unit.id~"][amount]" %}
+    {% set hiddenInputName = "sylius_refund_units["~unit.id~"][partial-id]" %}
```
Comment (GSadee) [inline templates/admin/order/refund/content/sections/form/items/table/body/products/refund_value.html.twig]: Use snake_case here

**Review by mpysiak — APPROVED** [review]

File: config/app/twig_hooks/admin/credit_memos/show.yaml
Comment (GSadee) [reply config/app/twig_hooks/admin/credit_memos/show.yaml:1]: This has not been solved

File: config/app/twig_hooks/admin/order/show.yaml [hunk 4]
```diff
@@ -19,48 +19,48 @@ sylius_twig_hooks:
 
         'sylius_admin.order.show.content.sections.payments.refund_header':
             title:
-                template: '@SyliusRefundPlugin/order/admin/content/sections/payments/header/title.html.twig'
+                template: '@SyliusRefundPlugin/admin/order/content/sections/payments/header/title.html.twig'
                 priority: 100
```
Comment (GSadee) [inline config/app/twig_hooks/admin/order/show.yaml]:
```suggestion
                priority: 0
```

File: config/app/twig_hooks/admin/order/show.yaml [hunk 5]
```diff
@@ -19,48 +19,48 @@ sylius_twig_hooks:
...
             actions:
-                template: '@SyliusRefundPlugin/order/admin/content/sections/payments/item/actions.html.twig'
+                template: '@SyliusRefundPlugin/admin/order/content/sections/payments/item/actions.html.twig'
                 priority: 0
 
         'sylius_admin.order.show.content.sections.payments.refund_item.actions':
             complete:
-                template: '@SyliusRefundPlugin/order/admin/content/sections/payments/item/actions/complete.html.twig'
+                template: '@SyliusRefundPlugin/admin/order/content/sections/payments/item/actions/complete.html.twig'
                 priority: 100
```
Comment (GSadee) [inline config/app/twig_hooks/admin/order/show.yaml]:
```suggestion
                priority: 0
```

File: config/app/twig_hooks/shop/account/order/show.yaml [hunk 6]
```diff
@@ -2,11 +2,11 @@ sylius_twig_hooks:
...
-                template: '@SyliusRefundPlugin/order/shop/credit_memos.html.twig'
+                template: '@SyliusRefundPlugin/shop/order/credit_memos.html.twig'
                 priority: -100
 
         'sylius_shop.account.order.show.content.main.credit_memos':
             list_action_buttons:
-                template: '@SyliusRefundPlugin/order/shop/download_button.html.twig'
+                template: '@SyliusRefundPlugin/shop/order/download_button.html.twig'
                 enabled: '%sylius_refund.pdf_generator.enabled%'
                 priority: 100
```
Comment (GSadee) [inline config/app/twig_hooks/shop/account/order/show.yaml]:
```suggestion
                priority: 0
```

File: config/app/twig_hooks/admin/order/show.yaml [hunk 7]
```diff
@@ -19,48 +19,48 @@ sylius_twig_hooks:
...
 
         'sylius_admin.order.show.content.sections.credit_memos.header':
             title:
-                template: '@SyliusRefundPlugin/order/admin/content/sections/credit_memos/header/title.html.twig'
-                priority: 100
+                template: '@SyliusRefundPlugin/admin/order/content/sections/credit_memos/header/title.html.twig'
+                priority: 0
 
-        'sylius_admin.order.show.content.sections.credit_memos.credit_memos.item.actions':
+        'sylius_admin.order.show.content.sections.credit_memos.credit_memos.items.actions':
```
Comment (GSadee) [inline config/app/twig_hooks/admin/order/show.yaml:62]: `credit_memos` part is duplicated

File: templates/admin/order/credit_memo/list.html.twig [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+                        <td class="w-1 py-2">
+                            <div class="d-flex flex-column">
+                                <strong>{{ credit_memo.issuedAt|date('d-m-Y') }}</strong>
+                                <div class="text-secondary">{{ credit_memo.number }}</div>
+                            </div>
+                        </td>
+                        <td class="text-center">{% include '@SyliusAdmin/shared/grid/field/channel.html.twig' with {'data': credit_memo.channel} %}</td>
+                        <td class="text-end"><strong>{{ money.format(credit_memo.total, credit_memo.currencyCode) }}</strong></td>
+                        <td class="w-1">
+                            {% hook 'sylius_admin.order.show.content.sections.credit_memos.credit_memos.items.actions' %}
```
Comment (GSadee) [inline templates/admin/order/credit_memo/list.html.twig:19]: Why are we triggering this hook here? 🤔

File: templates/admin/order/refund/content/sections/form/items/table/body/shipment.html.twig [hunk 9]
```diff
@@ -19,10 +19,10 @@
         </td>
         <td class="aligned collapsing partial-refund">
 
-            <div class="ui labeled input">
-                <div class="ui label">{{ order.currencyCode|sylius_currency_symbol }}</div>
+            <div class="input-group">
+                <div class="input-group-text">{{ order.currencyCode|sylius_currency_symbol }}</div>
                 {% set inputName = "sylius_refund_shipments["~shipping_adjustment.id~"][amount]" %}
```
Comment (GSadee) [inline templates/admin/order/refund/content/sections/form/items/table/body/shipment.html.twig:24]: Use snake_case here

File: templates/admin/order/credit_memo/list.html.twig (hunk 8 above)
Comment (bartek-sek) [reply templates/admin/order/credit_memo/list.html.twig:19]: This is the template that will be refactored in the next task, so I left it without changes

File: templates/admin/order/refund/content/sections/form/items/table/body/shipment.html.twig (hunk 9 above)
Comment (bartek-sek) [reply templates/admin/order/refund/content/sections/form/items/table/body/shipment.html.twig:24]: This is the template that will be refactored in the next task, so I left it without changes

File: config/app/twig_hooks/admin/order/show.yaml (hunk 7 above)
Comment (bartek-sek) [reply config/app/twig_hooks/admin/order/show.yaml:62]: This hook is used in partial that will be refactored in the next task, so I left it without changes.

File: templates/admin/credit_memo/show/content/sections/memo_details/table/body/net_value.html.twig [hunk 10]
```diff
@@ -0,0 +1,5 @@
+{% set item = hookable_metadata.context.item %}
+
+<td>
+    {{ '%0.2f'|format(item.netValue/100) }}
```
Comment (Wojdylak) [inline templates/admin/credit_memo/show/content/sections/memo_details/table/body/net_value.html.twig:4]:
```suggestion
    {{ (item.netValue / 100)|number_format(2) }}
```

**Review by Wojdylak — APPROVED** [review]

File: templates/admin/order/credit_memo/list.html.twig (hunk 8 above)
Comment (GSadee) [reply templates/admin/order/credit_memo/list.html.twig:19]: Ok, but this hook has to be changed in the next iteration 🖖🏻

**Pull request merged**